src/Helpers/BuildLocal.php

```diff
@@ -58,9 +58,17 @@ public function __construct( $fileName, $physicalPath, $data=false ) {
 
                 $this->buildProp = explode( "\n", $propsFileContent );
 
+                if ( $tokens['date'] == '' ) {
+                    $timestamp = filemtime( $this->filePath );
+                } else {
+                    date_default_timezone_set('Pacific/Kiritimati'); // the earliest time zone on Earth UTC+14:00
```

Comment:
Shouldn't this be reset to the current used one once you finish to calculate the date later?